presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml

```diff
@@ -2,32 +2,37 @@ datasource: presto
 query-names: presto/tpcds/${query}.sql
 runs: 6
 prewarm-runs: 2
-before-execution: sleep-4s, presto/session_set_reorder_joins.sql
+before-execution: sleep-4s
 frequency: 7
 database: hive
-tpcds_small: tpcds_10gb_orc
-tpcds_medium: tpcds_100gb_orc
-tpcds_large: tpcds_1tb_orc
+tpcds_10: tpcds_sf10_orc
+tpcds_30: tpcds_sf30_orc
+tpcds_100: tpcds_sf100_orc
+tpcds_300: tpcds_sf300_orc
+tpcds_1000: tpcds_sf1000_orc
+tpcds_3000: tpcds_sf3000_orc
+tpcds_10000: tpcds_sf10000_orc
```

Comment:
inline

Reply:
done

Reply:
squash the inline commit with this

Reply:
it looks like you now removed inlining

Reply:
this cannot be written directly, it needs to go via a property, it's because of the underlying mechanism